python/setup.py

```diff
@@ -255,6 +255,11 @@ def get_packages(self):
         "scipy",
     ]
 
+    # Ray AI Runtime should encompass Data, Tune, and Serve.
+    setup_spec.extras["air"] = (
+        setup_spec.extras["tune"] + setup_spec.extras["data"] + setup_spec.extras["serve"]
+    )
```

Comment:
I would recommend doing the list(set(... )) trick to deduplicate (see below). Tune and Serve have overlapping dependencies.